Update create-ilb-ase.md

rajanbhayana opened the pull request:
I have done a prototype to support my note and making a separate blog for same.

azurecla commented:
Hi __@jpsnover__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@jpsnover : Thanks for your contribution to the Azure documentation! The author, @ErikjeMS, has been notified to review your proposed change.

ErikjeMS commented:
@jpsnover Thanks for the change!

#sign-off

rajanbhayana committed:
Update create-ilb-ase.md

I have done a prototype to support my note and making a separate blog for same.
